app/controllers/public_revision_notification_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class PublicRevisionMailer < ApplicationMailer
+  def public_revision_notification(revision)
+    @revision = article_revision
+    mail(from: @revision.person.email, to: @revision.person.email, subject: 'Nowa wersja artykułu')
```

Comment:
W `ArticleRevision` dodać metodę zwracającą email redaktora `editor_email`. Pamiętać o ewnetualnych błędach - jeżli email jest pusty, to nie wysyłamy.